README.md

```diff
@@ -1,6 +1,24 @@
 # NLPpreprocessing
 A comprehensive NLP preprocessing package for clinical notes sentence boundary detection, tokenization
 
+## install
```

Comment:
Should we add the recommended Python version here?

Reply:
add python version